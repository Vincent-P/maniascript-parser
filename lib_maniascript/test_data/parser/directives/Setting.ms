#Setting S_FastRespawn True as _("Fast respawn on the checkpoint")
