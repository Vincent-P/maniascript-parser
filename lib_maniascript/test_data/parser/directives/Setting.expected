NODE_ROOT 0..66 {
  NODE_SETTING 0..66 {
    TOKEN_SETTING("#Setting") 0..8
    TOKEN_WHITESPACE(" ") 8..9
    NODE_IDENTIFIER 9..22 {
      TOKEN_IDENT("S_FastRespawn") 9..22
    }
    TOKEN_WHITESPACE(" ") 22..23
    NODE_BINARY_OP 23..66 {
      NODE_IDENTIFIER 23..27 {
        TOKEN_IDENT("True") 23..27
      }
      TOKEN_WHITESPACE(" ") 27..28
      TOKEN_AS("as") 28..30
      TOKEN_WHITESPACE(" ") 30..31
      NODE_FUNCTION_CALL 31..66 {
        NODE_IDENTIFIER 31..32 {
          TOKEN_IDENT("_") 31..32
        }
        TOKEN_OPEN_PAREN("(") 32..33
        NODE_STRING 33..65 {
          TOKEN_STRING_START("\"") 33..34
          TOKEN_STRING_CONTENT("Fast respawn on the checkpoint") 34..64
          TOKEN_STRING_END("\"") 64..65
        }
        TOKEN_CLOSE_PAREN(")") 65..66
      }
    }
  }
}